### DISCUSSION TITLE
这种东西就很难受，不过 Apache 也没有什么办法……

### DISCUSSION THREAD

**Discussion by RtYkk** [barry-ran/QtScrcpy#376] [General]
https://cn.mirroid.com/ 看起来就是本 repo 的复刻。
不过里面的转发声音的解决方案我觉得比较有意思，比 TODO 里面的 [sndcpy](https://github.com/rom1v/sndcpy) 好，因为 sndcpy 必须没有 [App Restirctions](https://github.com/rom1v/sndcpy#apps-restrictions) 
sndcpy 去年我试过一次，果然没有成功。它里面说的那个 [Bluetooth Audio Receiver](https://www.microsoft.com/en-us/p/bluetooth-audio-receiver/9n9wclwdqs5j?activetab=pivot:overviewtab) ，我觉得效果还是不错的。

**Comment by barry-ran** [comment]
他们用的什么方案

**Reply by RtYkk** [reply]
> 它里面说的那个 [Bluetooth Audio Receiver](https://www.microsoft.com/en-us/p/bluetooth-audio-receiver/9n9wclwdqs5j?activetab=pivot:overviewtab)

一个第三方 app，相当于电脑变蓝牙音箱。好像不太优雅……但是限制应该少些，只要蓝牙能输出的声音那个 app 就都能传输
